### ISSUE TITLE
Crowd funding compatible license

### ISSUE THREAD

**Issue by aquabu** [aquabu/peer-production-royalty-token-license#4]
Can one license support this or does there need to be one for **equity** and the other for **royalties**?

**Comment by jesseposner** [comment]
I think it would better to have separate agreements for equity and royalties because it's very important that both users and regulators understand that the royalty agreement is not intended to be subject to securities regulation, whereas the equity agreement would be, and so a firm separation would help underscore the distinction.

**Comment by aquabu** [comment]
Yes. Maybe I should take the leap and rename the repository to Dynamic Royalty Organization. Naming is hard.

**Comment by jesseposner** [comment]
@aquabu From a certain perspective, it's not necessarily an "organization" because there's no equity and no specific jurisdiction. Maybe something like the following would work:
- Dynamic Royalty Licensing
- Dynamic Royalty Software Development
- Dynamic Royalty Open-Source Software

**Comment by aquabu** [comment]
Dynamic Royalty Licensing - this could work
Dynamic Royalty Software Development - license could and may to apply to co-authored books.
Dynamic Royalty Open-Source Software - it may be closed source

Which of these are ok?
1. Dynamic Royalty Token
2. Dynamic Royalty Coin
3. Dynamic Share
4. Dynamic Royalty Share

**Comment by jesseposner** [comment]
I like Dynamic Royalty Token, because a token is something given as a guarantee of a right, in this case the royalty rights of the contributors. I prefer token to coin or share because a coin implies currency and a share implies equity.
